lib/io/event/selector/select.rb

```diff
@@ -326,8 +290,8 @@ def select(duration = nil)
 				writable = Array.new
 				priority = Array.new
 				
-				@waiting.each do |io, waiter|
-					waiter.each do |fiber, events|
+				@waiting.each do |io, h|
```

Comment:
I personally try to avoid using `_name` to indicate unused, I'm not against it, but I also don't like the edit amplification it sometimes causes.

I would suggest calling `h` -> `fibers`?

Reply:
Alright.